dygraph/mobilenet_v2/mobilenet_v2.py

```diff
@@ -0,0 +1,366 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
```

Comment:
The image classification code can use the same training code, so I think you can spilt the network architecture code and training code so that all the models can reuse it.